### PR TITLE
Setup initial serial comms to raspi

### PR DESCRIPTION
**Pull request by andrew103**
Right now this just "ping"s with a single value and "pong"s when the raspi sends it back to acknowledge the successful connection

### CODE REVIEW

File: ard_controller/ard_controller.ino [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  // put your setup code here, to run once:
+  Serial.begin(115200);
+  Serial1.begin(115200);
+
+  Timer1.attachInterrupt(serial_ISR).setFrequency(1).start();
+}
+
+void loop() {
+  int data;
+  if (Serial1.available() > 0) {
```
Comment (goobta) [inline ard_controller/ard_controller.ino:15]: This ping/pong behavior only happens when any data is sent and received--aka no parsing is being done?

File: ard_controller/ard_controller.ino [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#include <DueTimer.h>
+
+bool write_flag = false;
+
+void setup() {
+  // put your setup code here, to run once:
+  Serial.begin(115200);
+  Serial1.begin(115200);
```
Comment (goobta) [inline ard_controller/ard_controller.ino:8]: this is cause I haven't touched arduino in forever, but are `Serial` and `Serial1` just magical global variables?

File: ard_controller/ard_controller.ino [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  Timer1.attachInterrupt(serial_ISR).setFrequency(1).start();
```
Comment (goobta) [inline ard_controller/ard_controller.ino:10]: just another question out of curiosity: does this function just repeat the pings until a pong responds once?

**Review by goobta — APPROVED** [review]
just some questions for my curiosity, but the pr lgtm

File: ard_controller/ard_controller.ino (hunk 2 above)
Comment (andrew103) [reply ard_controller/ard_controller.ino:8]: Yep

File: ard_controller/ard_controller.ino (hunk 3 above)
Comment (andrew103) [reply ard_controller/ard_controller.ino:10]: It sends a ping every second regardless of if a pong is received

File: ard_controller/ard_controller.ino (hunk 1 above)
Comment (andrew103) [reply ard_controller/ard_controller.ino:15]: Correct. It doesn't look at the data, just checks for data being transmitted/received

Comment (goobta) [reply ard_controller/ard_controller.ino:15]: ack

File: ard_controller/ard_controller.ino (hunk 2 above)
Comment (goobta) [reply ard_controller/ard_controller.ino:8]: ack

File: ard_controller/ard_controller.ino (hunk 3 above)
Comment (goobta) [reply ard_controller/ard_controller.ino:10]: ah okok makes sense

**Pull request merged**